expression/builtin_math_vec.go

```diff
@@ -1070,7 +1070,8 @@ func (b *builtinSignSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column) er
 }
 
 func (b *builtinConvSig) vectorized() bool {
-	return true
+	// TODO: change the vecEval match hybrid type fixing. Then open the vectorized evaluation.
+	return false
```

Comment:
would you like to create an issue for this?

Reply:
I will update https://github.com/pingcap/tidb/issues/12106, since the PR merged.